docs/logagent/releasenotes.md

```diff
@@ -0,0 +1,83 @@
+title: Logagent Release Notes
+description: Features and improvements releases for Sematext Logagent
+
+Logagent is a dynamic project, and releases are typically in a high frequency. 
+We list here just important changes, new features, and critical bug fixes. 
```

Comment:
We list here just important changes, new features, and critical bug fixes.  => This Changelog contains only important changes, new features, and critical bug fixes.